### DISCUSSION TITLE
Output folder name (android and ios)

### DISCUSSION THREAD

**Discussion by vendramini** [expo/expo#28247] [Archived: Questions]
Hello people :)

I'm using the same project to compile for different brands, only changing images assets and colors. I'm picking up the correct theme during build-time in `tailwind.config.js` and `app.config.ts`.

Is it possible to change the output folders? Instead of `android` and `ios` could it be `android-{brand}` and `ios-{brand}`?

**Comment by tkrafael** [comment]
Same question here. I have two apps using similar code, one of which is for employee and other for customer. Everytime I need to test one or another, need to rebuild everything

**Comment by tkrafael** [comment]
What I did so far:

1. Build employee App as usual
2. move ios folder to ios-employee
3. create a symlink pointing ios-employee to ios folder:
```
# in linux/mac:
ln -s ios-employee ios
# windows
ln -s ios ios-employee
```
4. Do the same thing to customer, pointing to correct folder.

Some caveats:
- If ios folder is empty, it asks to reinitialize folder. In that case expo will delete folder and recreate, losing symbolic link. You must have target ios folder containing at least part of the previous build.
- untested in android

**Comment by tkrafael** [comment]
Didn't work. As soon as app tries to start, it errors out on device. Gave up using this approach. Maybe maintain two folders and manually rename it

**Comment by axelmy318** [comment]
I'd love to see this feature added. Also need it for kind of the same reasons.

**Comment by axelmy318** [comment]
Bumping this ticket.

**Comment by BeemWex** [comment]
I also need a solution to this - with 5+ different themes for a whitelabel app deleting the build folders and rebuilding every time we want to test a different variant has become our biggest timesink.
